src/main/jbake/content/docs/what.adoc

```diff
@@ -0,0 +1,208 @@
+= Why Hawkular?
+Heiko Rupp
+2015-07-04
+:description: Vision for Hawkular
+:jbake-type: page
+:jbake-status: draft
+
+
+== Why Hawkular?
+
+.NOTE: WHY: what is the purpose, why Hawkular
+
+To be the best tool to monitor JBoss and other Java Middleware helping
+you spending less and more enjoyable time on middleware management.
+
+OR:
+
+Hawkular will reduce the effort and time spent on systems management and monitoring
+and make this task more enjoyable.
+
+
+.NOTE: How
+
+This is done through automation of various aspects like resource discovery, root cause analysis,
+predictive analytics and easy to use user interfaces.
+
+
+== The Goals of Hawkular
+
+Hawkular is clearly a project to best monitor and manage JBoss Projects as well as other Java processes.
+
+For managed objects that Hawkular knows about, it will provide a specific, _easy to use_ environment.
+Hawkular will center around the notion of _applications_ that is the unit of interest in an operating environment.
+
+Of course the tool can only be as good as the understanding of the managed objects, so it will use
+_graceful degradation_ to provide a very good experience for well understood targets and a more generic
+one for others. Managed platforms will be able to supply parts of the user interface to also provide a specific
+interface.
+
+Hawkular will be easy to integrate into existing tools landscapes. This includes access in various programming
```

Comment:
Does that mean both for input of data as well as output? Will I be able to send data to other monitoring solutions?